pkg/cache/cache.go

```diff
@@ -90,6 +90,8 @@ type Informer interface {
 type ObjectSelector internal.Selector
 
 // SelectorsByObject associate a client.Object's GVK to a field/label selector.
+// Use SelectorsByObjectDefaultKey as key to specify a default selector that
+// will be used for all types that do not have a selector configured.
```

Comment:
Is this comment still valid?

Reply:
err after the back-and-forth regarding magic key vs config option I decided to use the magic key, as it is easy to add a config option later but not easy to remove it again: https://github.com/kubernetes-sigs/controller-runtime/pull/1710#issuecomment-962218495

Apparently I forgot to update the code though :grimacing: did that now.

I ended up not re-using the `ObjectAll` because it is a default that doesn't apply to everything, just to everything that doesn't have a setting so the name might cause confusion.

Reply:
> I ended up not re-using the ObjectAll because it is a default that doesn't apply to everything, just to everything that doesn't have a setting so the name might cause confusion.

Emm.. I'm confused that the **default** and **all** here seem to be the same semantics? They all mean the default configuration for all kinds of object, except the objects that have their specific values.